src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -494,7 +494,7 @@ export function getMultipleGeneResultKey(groupedOql: MergedTrackLineFilterOutput
 export function makeEnrichmentDataPromise<T extends {hugoGeneSymbol:string, pValue:number, qValue?:number}>(params:{
     store?:ResultsViewPageStore,
     await: MobxPromise_await,
-    getSelectedProfile:()=>MolecularProfile|undefined,
+    getSelectedProfiles:()=>{[id:string]:MolecularProfile},
```

Comment:
maybe getSelectedProfilesMap since it's not an array.  plural (s) implies array to me.